api/v1alpha3/machine_types.go

```diff
@@ -219,15 +219,6 @@ type Bootstrap struct {
 	// +optional
 	ConfigRef *corev1.ObjectReference `json:"configRef,omitempty"`
 
-	// Data contains the bootstrap data, such as cloud-init details scripts.
```

Comment:
I don't think we want to remove anything in the v1alpha3 types

Reply:
https://github.com/kubernetes-sigs/cluster-api/commit/ed9029bf32c3d2650703030f1783a21e7664653e already removed Data from v1alpha4

Reply:
+1

Reply:
Got it, thanks for the clarification!